v0/util/index.js

```diff
@@ -953,6 +953,14 @@ function returnArrayOfSubarrays(arr, len) {
   return chunks;
 }
 
+// Helper method to add external Id to traits
+// Traverse through the possible keys for traits using generic mapping and add externalId if traits found
+function addExternalIdToTraits(message) {
+    const identifierType = message.context.externalId[0].identifierType;
+    const identifierValue = message.context.externalId[0].id;
+    set(getFieldValueFromMessage(message, "traits"), identifierType, identifierValue);
```

Comment:
Should we check the value for null/undefined/falsy values before setting it?

Reply:
set method is already making checks for undefined, nulls etc., It should suffice in this case. Please let me know your opinion